.github/workflows/CI.yml

```diff
@@ -163,7 +163,7 @@ jobs:
             backend: 'OPENMP'
             cmake_build_type: 'Release'
             kokkos_ver: '4.3.00'
-            arborx: 'fee3d777c0325a90ab50671fca69f2dc98f2a882'
+            arborx: 'v2.0'
```

Comment:
Doesn't arborx 2.0 require Kokkos 4.5?

Reply:
You are right. I switched Kokkos versions.